source/transfer_list.rst

```diff
@@ -42,6 +42,9 @@ Transfer list requirements
 
 **R4:** When relocating the TL, the offset from `tl_base_pa` to the nearest alignment boundary specified by the `alignment` field in the TL header must be preserved.
 
+**R5:** If a TE type definition contains a sub-type field, then multiple entries of that TE type are allowed in the TL. The TE must have unique values in the sub-type field.
```

Comment:
perhaps *a* sub-type, rather than *the* ?

This seems reasonable to me, but I can think of a case where a subtype per se might not be needed.

If each entry has a memory range, then the range itself (e.g. start and size) would be enough to distinguish the entries. But still, I think the language here is an improvement, so I am happy enough with it.